pkg/ccl/backupccl/restore_job.go

```diff
@@ -1169,7 +1170,11 @@ func (r *restoreResumer) Resume(
 
 	numClusterNodes, err := clusterNodeCount(p.ExecCfg().Gossip)
```

Comment:
Should we also move this call to closer to the telemetry since that's the only thing that it's used for?

Reply:
I sorta considered that but then we'd need to pass Gossip into `restore` too. I think the _real_ answer is that we should be using number of nodes we actually plan the flow on, so I think this should eventually go away rather than move. In the interest of keeping this backport-friendly I think just leave it as is until then?